frame/session/src/tests.rs

```diff
@@ -336,7 +336,7 @@ fn session_keys_generate_output_works_as_set_keys_input() {
 }
 
 #[test]
-fn return_true_if_more_than_third_is_disabled() {
+fn disable_index_returns_true_if_already_disabled() {
```

Comment:
Do you "returns_false_if_already_disabled" (and true if not disabled and in the set) ?

Reply:
Indeed, good catch.